src/components/ebay-pagination/test/mock/index.js

```diff
@@ -79,6 +105,78 @@ exports.Links_9Items_7Selected = assign({}, exports.Base_0Items, {
     )
 });
 
+exports.Links_16ItemsDots_1Selected = assign({}, exports.Base_0Items_Dots, {
```

Comment:
Is there a reason we're not using ES6 object spread instead of assign? I thought we could use ES6 everywhere besides the .marko files. 

Maybe there's some background info I'm misssing?